clang/lib/Sema/SemaChecking.cpp

```diff
@@ -7795,6 +7795,20 @@ bool Sema::CheckIntelSYCLPtrAnnotationBuiltinFunctionCall(unsigned BuiltinID,
   return false;
 }
 
+static llvm::APSInt getSYCLAllocaDefaultSize(const ASTContext &Ctx,
+                                             const VarDecl *VD) {
+  assert(VD && "Expecting valid declaration");
```

Comment:
Nit: You don't need this assert since the cast to `VarDecl ` should technically catch this but I guess it makes sense to keep it if you ever expect to call this from elsewhere.